mne/viz/_3d.py

```diff
@@ -864,42 +854,30 @@ def plot_dipoles(dipoles, trans, subject, subjects_dir=None,
 
     points, faces = read_surface(fname)
 
-    pos = np.zeros((n_dipoles, n_times, 3))
-    ori = np.zeros((n_dipoles, n_times, 3))
-    amp = np.zeros((n_dipoles, n_times))
+    pos = np.zeros((n_dipoles, 3))
+    ori = np.zeros((n_dipoles, 3))
     for i_dip in range(n_dipoles):
-        pos[i_dip], ori[i_dip] = dipoles[i_dip].pos, dipoles[i_dip].ori
-        amp[i_dip] = dipoles[i_dip].amplitude
-
-    from mayavi import mlab
-    import matplotlib.pyplot as plt
-    from matplotlib.colors import ColorConverter
-    color_converter = ColorConverter()
+        pos[i_dip], ori[i_dip] = dipoles[i_dip].pos[0], dipoles[i_dip].ori[0]
 
     fig = mlab.figure(size=fig_size, bgcolor=bgcolor, fgcolor=(0, 0, 0))
-    ax = plt.subplot()
 
     points *= 1e-03
-    points = apply_trans(coord_trans, points)
+    points = apply_trans(trans['trans'], points)
 
     mlab.triangular_mesh(points[:, 0], points[:, 1], points[:, 2],
                          faces, color=mesh_color, opacity=opacity)
 
     for i_dip, color in zip(range(len(dipoles)), colors):
         rgb_color = color_converter.to_rgb(color)
-        mlab.quiver3d(pos[i_dip, 0, 0], pos[i_dip, 0, 1], pos[i_dip, 0, 2],
-                      ori[i_dip, 0, 0], ori[i_dip, 0, 1], ori[i_dip, 0, 2],
+        mlab.quiver3d(pos[i_dip, 0], pos[i_dip, 1], pos[i_dip, 2],
+                      ori[i_dip, 0], ori[i_dip, 1], ori[i_dip, 2],
                       opacity=1., mode=mode, color=rgb_color,
                       scalars=dipoles[i_dip].amplitude.max(),
                       scale_factor=scale_factor)
 
-        # Plot the time-series
-        ax.plot(dipoles[i_dip].times, dipoles[i_dip].amplitude,
-                color=color, linewidth=1.5)
-    ax.set_ylabel('amplitude (nAm)')
-    ax.set_xlabel('times (ms)')
-
```

Comment:
Hum I had not noticed

Can you add an option so we see time courses in music example?



> Le 20 avr. 2015 à 20:40, Eric Larson <notifications@github.com> a écrit :
> 
> In mne/viz/_3d.py:
> 
> >                        opacity=1., mode=mode, color=rgb_color,
> >                        scalars=dipoles[i_dip].amplitude.max(),
> >                        scale_factor=scale_factor)
> >  
> > -        # Plot the time-series
> > -        ax.plot(dipoles[i_dip].times, dipoles[i_dip].amplitude,
> > -                color=color, linewidth=1.5)
> > -    ax.set_ylabel('amplitude (nAm)')
> > -    ax.set_xlabel('times (ms)')
> > -
> I removed the time plot aspect of this plotting function. I think we should probably later add a dipole.plot_timecourse or something for this. Combining them seems a bit clunky, especially since one is Mayavi and the other is matplotlib.
> 
> —
> Reply to this email directly or view it on GitHub.
>